tauri/src/app/webview/wry.rs

```diff
@@ -218,8 +218,7 @@ impl ApplicationDispatcherExt for WryDispatcher {
         Some(wry_callbacks),
         custom_protocol.map(|p| wry::CustomProtocol {
           name: p.name.clone(),
-          // TODO: Map error type instead of unwraping
-          handler: Box::new(move |a| Ok((*p.handler)(a).unwrap())),
+          handler: Box::new(move |a| (*p.handler)(a).map_err(|_| wry::Error::InitScriptError)),
```

Comment:
Shouldn't we add a TODO for including the path of the asset so the error message is more helpful?

Reply:
wry doesn't have any kind of error handling on custom protocol right now, so the error doesn't matter; it should probably eval a `console.error` on the webview. Can you look into that or maybe open an issue?

Reply:
On linux at least it gives you a pretty nice error on the Webview, I assume other platforms will do the same as it will be just like a failed network request to this can be resolved.
![image](https://user-images.githubusercontent.com/12448243/109098567-756ba580-76d6-11eb-852d-7e7862e0fc65.png)